Essentials/src/com/earth2me/essentials/commands/Commandnuke.java

```diff
@@ -25,10 +25,8 @@ protected void run(final Server server, final CommandSource sender, final String
         Collection<Player> targets;
         if (args.length > 0) {
             targets = new ArrayList<>();
-            int pos = 0;
-            for (String arg : args) {
-                targets.add(getPlayer(server, sender, args, pos).getBase());
-                pos++;
+            for (int i = 0; args.length > i; i++) {
```

Comment:
```suggestion
            for (int i = 0; i < args.length; ++i) {
```

Reply:
In response to the commit message `pop wanted this idk why tho`: The way I've suggested it is more commonly used (I don't think I've seen it the other way often if at all in Essentials, or most other code). Late night me reviewing this thought this was a bug at first because it was the other way around.